src/olympia/scanners/models.py

```diff
@@ -387,3 +421,14 @@ class Meta(AbstractScannerResult.Meta):
 class ScannerQueryMatch(ModelBase):
     result = models.ForeignKey(ScannerQueryResult, on_delete=models.CASCADE)
     rule = models.ForeignKey(ScannerQueryRule, on_delete=models.CASCADE)
+
+
+class VersionScannerFlags(ModelBase):
+    version = models.OneToOneField(
+        'versions.Version', primary_key=True, on_delete=models.CASCADE
+    )
+    needs_human_review_by_mad = models.BooleanField(default=False)
```

Comment:
I don't have anything better to suggest yet, but I'm not a huge fan of this name.

Reply:
Yep. I was thinking that, given that we do not have a lot of scanners, we could have one field per scanner in the future (and stop relying on the version fields in some cases)

Reply:
Yeah, that's not my concern. It's mostly the `by_` that I don't like: It kinda implies the version needs a human review to be done by the mad scanner, at least that how I read it the first time and I was confused :)

Reply:
oh sorry, I misunderstood.